python/plugins/processing/algs/lidar/fusion/ClipData.py

```diff
@@ -73,8 +90,5 @@ def processAlgorithm(self, progress):
         commands.append(extent[1])
         commands.append(extent[3])
         FusionUtils.runFusion(commands, progress)
-        commands = [os.path.join(FusionUtils.FusionPath(), 'LDA2LAS.exe')]
-        commands.append(outFile)
-        commands.append(self.getOutputValue(self.OUTPUT))
```

Comment:
if there is no need to convert ot LAS anymore, the two lines below should be removed as well